hw/ip/pwrmgr/data/pwrmgr_sec_cm_testplan.hjson

```diff
@@ -25,81 +25,187 @@
   testpoints: [
     {
       name: sec_cm_bus_integrity
-      desc: "Verify the countermeasure(s) BUS.INTEGRITY."
+      desc: '''Verify the countermeasure(s) BUS.INTEGRITY.
+            This entry is covered by tl_access_test
+            (hw/dv/tools/dvsim/tests/tl_access_tests.hjson)
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_tl_intg_err"]
     }
     {
       name: sec_cm_lc_ctrl_intersig_mubi
-      desc: "Verify the countermeasure(s) LC_CTRL.INTERSIG.MUBI."
+      desc: '''Verify the countermeasure(s) LC_CTRL.INTERSIG.MUBI.
+
+            **Stimulus**:
+            - Use comprehensive stimulus - reset and wakeup -
+              as background traffic to ensure this counter measure
+              is valid for various states of fast and slow state.
+            - Drive lc_hw_debug_en_i and lc_dft_en_i with
+              mixed valid and invalid values.
+
+            **Check**:
+            - Collect coverage by binding cip_mubi_cov_if to
+              tb.dut.lc_hw_debug_en_i and tb.dut.lc_dft_en_i
+            - Add assertion to check whether rom_intg_chk_dis
+              is set to '1' only when lc_dft_en_i or lc_hw_debug_en_i
+              is high.
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm_lc_ctrl_intersig_mubi"]
     }
     {
       name: sec_cm_rom_ctrl_intersig_mubi
-      desc: "Verify the countermeasure(s) ROM_CTRL.INTERSIG.MUBI."
+      desc: '''Verify the countermeasure(s) ROM_CTRL.INTERSIG.MUBI.
+
+            **Stimulus**:
+            - Use comprehensive stimulus - reset and wakeup -
+              as background traffic to ensure this counter measure
+              is valid for various states of fast and slow fsm.
+            - Drive rom_ctrl_i with mixed valid and invalid values.
+
+            **Check**:
+            - Collect coverage by binding cip_mubi_cov_if to
```

Comment:
do we need to check for correct fsm handling of invalid values also?

Reply:
I assume invalid value will be ignored by dut. Let me know if it is not the case.
I can add specific check on that.